### PR TITLE
[fix][doc] explicitly instruct clone the repo

### PR DESCRIPTION
**Pull request by tisonkun**
### Documentation

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)

- [ ] `doc-not-needed` 
(Please explain why)

- [x] `doc` 
(Your PR contains doc changes)

- [ ] `doc-complete`
(Docs have been already added)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,42 +20,30 @@
 -->
 
 # Pulsar C++ client library
-<!-- TOC depthFrom:2 depthTo:3 withLinks:1 updateOnSave:1 orderedList:0 -->
 
-- [Requirements](#requirements)
-- [Platforms](#platforms)
-- [Compilation](#compilation)
```
Comment (shibd) [inline README.md:27]: Why remove the catalog?

Comment (tisonkun) [reply README.md:27]: Read also https://github.com/apache/pulsar/pull/17470#discussion_r963277120

**Review by shibd — APPROVED** [review]

**Review by BewareMyPower — APPROVED** [review]

**Review by RobertIndie — APPROVED** [review]

**Comment by tisonkun** [comment]
Shall we merge this patch now? Or we're still waiting for specific reviewers' comment?

**Comment by BewareMyPower** [comment]
No. Just because it's not an urgent PR. It's ready to merge.

**Pull request merged**